src/ray/protobuf/gcs.proto

```diff
@@ -372,6 +372,9 @@ message GcsNodeInfo {
 
   // The death info of this node.
   NodeDeathInfo death_info = 29;
+
+  // The available resources of this node.
+  map<string, double> resources_available = 30;
```

Comment:
can we reuse `AvailableResources` -- https://github.com/ray-project/ray/blob/766fe52090d9e9d498500f9099c1db40d5a88195/src/ray/protobuf/gcs.proto#L270 ?

Reply:
It is slightly redundant because it will include the `node_id` twice. I'm not sure why `node_id` is embedded in the `AvailableResources` message...